scripts/ci-test-orchestrator.js

```
@@ -0,0 +1,385 @@
+    return new Promise((resolve) => {
+      const args = command.split(' ');
+      const cmd = args.shift();
+      
+      console.log(`   ${isFallback ? 'Fallback ' : ''}Command: ${command}`);
+      
+      const process = spawn(cmd, args, {
+        stdio: 'pipe',
+        shell: true,
+        env: {
+          ...process.env,
+          CI: this.config.isCI ? 'true' : process.env.CI,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)